src/commands/main.ts

```diff
@@ -230,5 +242,11 @@ export const createMainCommand = () => {
     })
     .action(mainCommand)
 
+  program.commands.forEach((cmd) => {
```

Comment:
This is where we add the `--force` option to each command that is in the set above. This allows the `--force` option to be unified across all commands, for adding the `--force` flag in ci/cd environments.